server/routes/api/cards.js

```diff
@@ -44,71 +43,34 @@ router.post(
 // @desc Move Card to another column
 // @access private
 
-router.put("/:id", auth, (req, res) => {
+router.put("/:id", auth, async (req, res) => {
   try {
-    console.log(req.body);
-    Card.findByIdAndUpdate(req.params.id, req.body, (err, updatedCard) => {
-      if (!updatedCard) return res.status(400).send({ msg: "Invalid Card" });
-
-      console.log("updatedCard", updatedCard);
-      res.send(updatedCard);
+    const { name, details } = req.body;
+    await Card.findByIdAndUpdate(req.params.id, {
+      $set: {
+        name: name,
+        details: details,
+      },
     });
+    res.status(200).send();
   } catch (err) {
-    res.status(500).send("Server Error");
-  }
-});
-
-// @route PUT api/cards/update
-// @desc Move Card to another column
-// @access Public
-router.put("/:id", async (req, res) => {
-  try {
-    const index = req.body.index;
-    const newColId = req.body.newCol;
-    const oldColId = req.body.oldCol;
-    const cardId = req.params.id;
-
-    //Have to update column where card is removed,
-    const oldCol = await Column.findById(oldColId);
-    const oldColCards = await oldCol.cards;
-    await Column.findByIdAndUpdate(oldColId, {
-      cards: oldColCards.filter((c) => c !== cardId)
-    });
-
-    //Update column where card is added
-    const newCol = await Column.findById(newColId);
-    let newColCards = await newCol.cards;
-    await Column.findByIdAndUpdate(newColId, {
-      cards: [
-        ...newColCards.slice(0, index),
-        cardId,
-        ...newColCards.slice(index)
-      ]
+    console.error(err);
+    res.status(500).json({
+      errors: { CARD_NOT_SAVED: "Could not save card." },
```

Comment:
Good call on proactively deleting the deprecated function.

Reply:
:)